### DISCUSSION TITLE
Its unclear how to load Forth files from disk and `v` lacks the `:e <loadfile>` command

### DISCUSSION THREAD

**Discussion by jeffgazso** [jkotlinski/durexforth#570] [Q&A] [answered]
Using the *v5.0.0-preview* release I can write files to disk just fine from within *v*; the `:w <save_file>` command works as expected. Unfortunately, it seems that `:e <load_file>` command is unimplemented. Consequently, it's a bit unclear how to load a file from disk. Looking at the documentation, I see that there is a `save-forth` word, but not a `load-forth`. I can see my files just fine with `ls`, I just can't seem to load them. What am I missing?

Tangential but probably related, when I extract a Forth source file from a `.d64` image and look at it with a hex editor such as `xxd`, I see that there is what appears to be a two byte load address. Is this a dummy value, or does it have some significance, as it does for tokenized BASIC files? I'm kind of surprised Forth source files aren't stored as SEQ on the CBM file system.

**Comment by Whammo** [comment, marked as answer]
Simply exit the editor and then re-invoke it referencing the desired file.
 The source files are indeed .prg files. The load address is ignored. This is to take advantage of fast load carts and etc.
 Thanks for your interest, and please let me know if you're inclined for further elucidation.
